src/site/siteSlice.ts

```diff
@@ -110,6 +124,35 @@ export const transferSites = createAsyncThunk<
   return result;
 });
 
+export const addSiteNote = createAsyncThunk<SiteNote, SiteNoteAddMutationInput>(
+  'site/addSiteNote',
+  async (siteNote, _) => {
+    let result = await siteService.addSiteNote(siteNote);
+    return siteService.collapseSiteNote(result);
+  },
+);
+
+export const deleteSiteNote = createAsyncThunk<SiteNote, SiteNote>(
+  'site/deleteSiteNote',
+  async siteNote => {
+    let result = await siteService.deleteSiteNote(siteNote);
+    return result;
+  },
+);
+
+export const updateSiteNoteOld = createAsyncThunk(
+  'site/updateSiteNote',
+  siteService.updateSiteNote,
+);
```

Comment:
question: why is there both updateSiteNote and updateSiteNoteOld?

Reply:
Fixed!